mattermost-plugin/webapp/src/index.tsx

```diff
@@ -172,7 +172,14 @@ export default class Plugin {
             }
 
             if (rudderKey !== '') {
-                rudderAnalytics.load(rudderKey, rudderUrl)
+                const rudderCfg = {} as {setCookieDomain: string}
+                if (siteURL && siteURL !== '') {
+                    try {
+                        rudderCfg.setCookieDomain = new URL(siteURL).hostname
+                        // eslint-disable-next-line no-empty
+                    } catch (_) {}
```

Comment:
nit: given that we have the catch section, is there maybe something better that just ignoring? not sure if in this case is worth even doing some sort of logging or even tracking the failure somehow.
as mentioned before, it is a nit, so, feel free to don't even provide an elaborate answer to this comment.

Reply:
I went this way because when your reach `new URL`, siteURL cannot be empty and have been validated beforehand by the server, so there's no reason for `new URL` to throw an exception... but if it does for some reason, I don't want to interrupt anything, so we just don't set the value.

Reply:
ok, thanks!